src/configureWorkspace/configurePython.ts

```diff
@@ -86,8 +96,18 @@ async function genDockerFile(serviceName: string, target: PythonTarget, projectT
         throw new Error(localize('vscode-docker.configurePython.unknownProjectType', 'Unknown project type: {0}', projectType));
     }
 
+    let userSwitchLogic = lowRightsUser;
+    let userWarning = '';
+
+    if (ports?.find(p => p < 1024)) {
```

Comment:
```suggestion
    if (ports?.some(p => p < 1024)) {
```
Equivalent of C#'s `Any`